### PR TITLE
RemoteInvocationHandler.reportStats LogRecord’s could get mangled during serialization

### PR DESCRIPTION
**Pull request by jglick**
See https://github.com/jenkinsci/jenkins/pull/4621 for the original buglet. As of https://github.com/jenkinsci/jenkins/pull/4912, seems better to just avoid the problematic idiom.

### CODE REVIEW

File: src/main/java/hudson/remoting/RemoteInvocationHandler.java [hunk 1]
```diff
@@ -719,63 +718,63 @@ private void reportStats() {
             if (m15Std > 1 && 100 < m15Avg - 2 * m15Std) {
                 if (tStd > 1 && 100 < tAvg - 2 * tStd) {
                     logger.log(Level.SEVERE,
```
Comment (jglick) [inline src/main/java/hudson/remoting/RemoteInvocationHandler.java:720]: These are some sort of exotic conditions with no test coverage that I know of, so just hoping it works.

Comment (jglick) [reply src/main/java/hudson/remoting/RemoteInvocationHandler.java:720]: see #81

**Comment by jeffret-b** [comment]
Builds fine on my system. Let's see if we can get a build on CI.

**Review by jeffret-b — APPROVED** [review]
Looks good. Hopefully it will work well enough.

String.format() can be slow, but we shouldn't be doing it enough here to cause performance problems.

**Pull request merged**